bigquery/src/test/java/com/google/cloud/hadoop/io/bigquery/AbstractBigQueryIoIntegrationTestBase.java

```diff
@@ -128,37 +128,40 @@ public AbstractBigQueryIoIntegrationTestBase(InputFormat inputFormat) {
    * intended for BigQueryFactory and adding them as GCS-equivalent credential settings.
    */
   public static Configuration getConfigForGcsFromBigquerySettings(
-      String projectIdvalue, String testBucket) {
+      String projectIdValue, String testBucket) {
     TestConfiguration configuration = TestConfiguration.getInstance();
 
-    String bigqueryServiceAccount = configuration.getServiceAccount();
-    if (Strings.isNullOrEmpty(bigqueryServiceAccount)) {
-      bigqueryServiceAccount = System.getenv(BigQueryFactory.BIGQUERY_SERVICE_ACCOUNT);
+    String serviceAccount = configuration.getServiceAccount();
```

Comment:
why do we want to rename this variables?

Reply:
Because the same service account and private key is used for BQ and GCS configuration.